.devcontainer/devcontainer.json

```diff
@@ -1,79 +1,43 @@
-// For format details, see https://aka.ms/devcontainer.json. For config options,
```

Comment:
oh, this is not the config i had in mind, but we are at several versions of VSCode ahead in the future now&hellip;